### ISSUE TITLE
【feat】Does it already support spark 3.2 or above?

### ISSUE THREAD

**Issue by zyclove** [apache/carbondata#4346]
Is it compatible with spark 3.2 and above?

Is this project still being maintained and upgraded? I see there has been no update for more than 3 months.

Will there be any major feature changes in the future?

Thanks

**Comment by chenliang613** [comment]
Yes,  in dev mailing list ,  we are discussing, upgrade to Spark 3.5

**Comment by 1411248231** [comment]
When will it be compatible with spark3.5? 
Thanks

**Comment by 1411248231** [comment]
Is it compatible with hadoop 3.3 and above?
